src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int firstLeg;
+    private int secondLeg;
+    private int thirdLeg;

```

Comment:
2 sides are enough to describe right triangle, hypotenuse is not necessary

Reply:
kind of yes, but I generate the hypotenuse through the conditions, so that it would be correct